src/assets/scripts/client/ui/VideoMapController.js

```diff
@@ -0,0 +1,220 @@
+     *
+     * Should be run once only on instantiation
+     *
+     * @for TrafficRateController
+     * @method _setupHandlers
+     * @chainable
+     */
+    _setupHandlers() {
+        this._onAirportChangeHandler = this.onAirportChange.bind(this);
+
+        return this;
+    }
+
+    /**
+     * Enable event handlers
+     *
+     * should be run only once on instantiation
+     *
+     * @for TrafficRateController
+     * @method enable
+     * @chainable
+     */
+    enable() {
+        this._eventBus.on(EVENT.AIRPORT_CHANGE, this._onAirportChangeHandler);
+
+        return this;
+    }
+
+    /**
+     * Disable event handlers
+     *
+     * @for TrafficRateController
+     * @method disable
+     * @chainable
+     */
+    disable() {
+        this._eventBus.off(EVENT.AIRPORT_CHANGE, this._onAirportChangeHandler);
+
+        return this;
+    }
+
+    /**
+     * Returns whether the airport selection dialog is open
+     *
+     * @for TrafficRateController
+     * @method isDialogOpen
+     * @return {boolean}
+     */
+    isDialogOpen() {
+        return this.$dialog.hasClass(SELECTORS.CLASSNAMES.OPEN);
+    }
+
+    /**
+    * @for TrafficRateController
+    * @method toggleDialog
+    */
+    toggleDialog() {
+        this.$dialog.toggleClass(SELECTORS.CLASSNAMES.OPEN);
+    }
+
+    /**
+     * Rebuilds the dialog body when the airport is changed.
+     *
+     * @for TrafficRateController
+     * @method onAirportChange
+     */
+    onAirportChange() {
+        this._buildDialogBody();
+    }
+
+    /**
+     * Builds the dialog body
+     *
+     * @for TrafficRateController
+     * @method _buildDialogBody
+     */
+    _buildDialogBody() {
+        this.$dialogBody.empty();
+        const airportModel = AirportController.airport_get();
+        const mapNames = airportModel.mapCollection.getMapNames();
+        this._selectedMaps = airportModel.mapCollection.getVisibleMapNames();
+
+        for (const mapName of mapNames) {
+            const isChecked = this._selectedMaps.includes(mapName);
+            const $formElement = this._buildRow(mapName, mapName, isChecked, this._onChangeSelectedMaps);
+            this.$dialogBody.append($formElement);
+        }
+    }
+
+    /**
+     * Build form element
+     *
+     * @for VideoMapController
+     * @method _buildRow
+     * @param key {string}
+     * @param label {string}
+     * @param checked {boolean} initial value
+     * @param onChangeMethod {function}
+     * @return {jquery|HTML Element}
+     */
+    _buildRow(key, label, checked, onChangeMethod) {
+        const _inputTemplate = checked ? `<input class="form-checkbox" type="checkbox" name="${key}" checked/>` :
+            `<input class="form-checkbox" type="checkbox" name="${key}"/>`;
+        const template = `
+            <div class="form-element">
+                ${_inputTemplate}
+                <label class="form-label"> ${label}</label>
+            </div>`;
+        const $element = $(template);
```

Comment:
now that we have a DOM element, you could leverage JS to set the `checked` property.  Check me on this, but probably something like:
```js
$element.find('.form-checkbox').prop('checked', checked);
```

Reply:
@n8rzz good feedback. JavaScript is not my 'native' language and I am more or less muddling along, so this sort of nuance is a big help. Thanks.

Reply:
You're doing pretty damn good for someone just muddling along!